fluentbit/Dockerfile

```diff
@@ -17,37 +20,64 @@ RUN export GNUPGHOME="$(mktemp -d)" \
     && gpg --keyserver keyserver.ubuntu.com --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A \
     && gpg --export --armor 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A > ${GNUPGHOME}/RPM-GPG-KEY-Percona \
     && rpmkeys --import ${GNUPGHOME}/RPM-GPG-KEY-Percona \
+    && microdnf install -y findutils \
     && curl -Lf -o /tmp/percona-release.rpm https://repo.percona.com/yum/percona-release-latest.noarch.rpm \
     && rpmkeys --checksig /tmp/percona-release.rpm \
     && rpm -i /tmp/percona-release.rpm \
     && rm -rf "$GNUPGHOME" /tmp/percona-release.rpm \
     && rpm --import /etc/pki/rpm-gpg/PERCONA-PACKAGING-KEY \
-    && percona-release setup pdpxc-8.0.23
+    && percona-release enable pdpxc-8.0.23 release
 
 # install exact version of PS for repeatability
-ENV PERCONA_VERSION 8.0.23-14.1.el7
+ENV PERCONA_VERSION 8.0.23-14.1.el8
+ENV FLUENTBIT_VERSION 1.8.9-1
 
 RUN set -ex; \
-    microdnf install -y postgresql-libs shadow-utils yum-utils logrotate \
-    percona-xtradb-cluster-client-${PERCONA_VERSION} tar vim-minimal; \
-    curl -Lf -o /tmp/procps-ng.rpm http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/procps-ng-3.3.15-6.el8.x86_64.rpm; \
+    if [ "$FLUENTBIT_SOURCE" == 'fluentbit_repository' ]; \
```

Comment:
I do not understand why it is needed.

Reply:
This block was introduced to preserve possibility building docker image with fluentbit from repository, this helps when version changed